src/css/footer.scss

```diff
@@ -75,65 +114,106 @@ footer {
     .half-circle {
         // override gatsby-image inline style
         position: absolute !important;
-        width: 205px;
-    }
-
-    @media (min-width: #{$bp-large}) {
-        .footer-columns {
-            ul {
-                padding: 0 50px 30px;
-            }
-        }
-
-        .half-circle {
-            right: 0;
-            bottom: 50%;
-            transform: translateY(50%);
+        width: 100px;
+        height: 200px;
+        transform-origin: bottom left;
+        transform: rotate(-270deg) translateY(50%);
+        left: 50%;
+        bottom: 100px;
+
+        [aria-hidden] {
+            padding-bottom: 0 !important;
         }
     }
 
-    @media (max-width: #{($bp-large - 1)}) {
+    @media (min-width: #{$bp-medium}) {
         padding: 120px 0 240px;
 
-        .half-circle {
-            bottom: -102px;
-            left: 52%;
-            transform: rotate(90deg) translateY(30%);
-        }
-    }
-
-    @media (max-width: #{($bp-medium - 1)}) {
-        padding: 30px 0 240px;
         .footer-columns {
-            flex-wrap: wrap;
+            flex-direction: row;
 
             .footer-column-1 {
-                flex: 0 0 100%;
-            }
-
-            ul {
+                flex: 0 0 auto;
+                order: 0;
+                flex-direction: column;
                 padding: 0 30px 30px;
             }
 
             li {
-                a {
-                    font-size: 18px;
+                padding: 0;
+
+                a:not(.btn) {
+                    line-height: 1.45;
+                }
+
+                .email-btn {
+                    background-color: $black;
+                    color: #fff;
+                    border-color: #fff;
+                    font-size: 15px;
+                    font-weight: normal;
+                    padding: 9px 24px;
+                    margin: 1.5em 0 0;
+                }
+            }
+        }
+
+        .keep-logo {
+            height: 65px;
+            background-image: url(../images/Keep-logo-circle.svg);
+            background-repeat: no-repeat;
+            background-position: top left;
```

Comment:
Do we need to re-specify the logo info?

Reply:
There's a different logo based on whether it's mobile or desktop. I can get rid of the duplicate `background-repeat: no-repeat` rule though!

Reply:
Oh goodness, yeah that makes sense. Might be worth dropping a comment for that, they look very similar at first blush.